app/navigation/stack-param-lists.ts

```diff
@@ -16,7 +16,7 @@ export type RootStackParamList = {
   authenticationCheck: undefined
   authentication: {
     screenPurpose: AuthenticationScreenPurpose
-    isPinEnabled: boolean
+    isPinEnabled: boolean | undefined
```

Comment:
doesn't feel right for this to be undefined.

Reply:
yes this is unnecessary, good catch -- the `PhoneValidationStackParamList['authentication']` has the definition we need (to use in `phone-auth.tsx`).